MatomoTracker.php

```diff
@@ -1477,6 +1477,21 @@ public function setRequestTimeout($timeout)
         $this->requestTimeout = $timeout;
         return $this;
     }
+	
+	/**
+     * Sets the request method to POST, which is recommended when using setTokenAuth()
+     * to prevent the token from being recorded in server logs. Avoid using redirects
+     * when using POST to prevent the loss of POST values. When using Log Analytics,
+     * be aware that POST requests are not parseable/replayable.
+     *
+     * @param string $method
+     * @return $this
+     */
+    public function setRequestMethod($method)
```

Comment:
@silvershadowcc could we rename this method to `setRequestMethodForNonBulkRequests` since it doesn't apply to bulk requests? If there's some better name feel free to also use a different name.

Reply:
`setRequestMethodNonBulk` for the sake of brevity? (since it already states "request")

Reply:
Sounds great 👍